content/docs/guides/parallel-execution.md

```diff
@@ -0,0 +1,396 @@
+---
+title: Parallel Execution
+subtitle: Using multiple threads to reduce test times
+polyglot:
+ - java
+ - kotlin
+ 
+weight: 1800
+---
+
+Cucumber-JVM allows [parallel execution](https://cucumber.io/blog/announcing-cucumber-jvm-4-0-0/) across multiple threads since version 4.0.0. There are several options to incorporate this inbuilt feature in a Cucumber project.
+
+- [JUnit](/docs/guides/parallel-execution/#junit)
+- [TestNG](/docs/guides/parallel-execution/#testng)
+- [CLI](/docs/guides/parallel-execution/#cli)
+
+For each of these options, this tutorial will look at the project setup, configuration settings, execution commands.
+
+
+# JUnit
+Cucumber can be executed in parallel using **JUnit and Maven test execution plugins**. In JUnit the **feature files are run in parallel rather than scenarios**, which means **all the scenarios in a feature file will be executed by the same thread**. One can use either Maven Surefire or Failsafe plugin for executing the runners.
+
+- Create a Maven project in your favorite IDE. Add `cucumber-java8` and `cucumber-junit` **dependencies** to the `POM`.
+
+```shell
+<dependency>
+	<groupId>io.cucumber</groupId>
+	<artifactId>cucumber-java8</artifactId>
+	<version>{{% version "cucumberjvm" %}}</version>
+	<scope>test</scope>
+</dependency>
+<dependency>
+	<groupId>io.cucumber</groupId>
+	<artifactId>cucumber-junit</artifactId>
+	<version>{{% version "cucumberjvm" %}}</version>
+	<scope>test</scope>
+</dependency>
+```
+
+- Create a **parallel** folder in `src/test/resources` path and add the two feature files (`scenarios.feature` and `scenariooutlines.feature`) inside it.

```

Comment:
No configuration reason. Just a package name I thought would be appropriate. Will mention that folder can be named anything.

Reply:
Ok, thanks. That's what I thought, I just wanted to make sure. If it's not needed, maybe don't mention creating a folder at all, to prevent confusion?

Reply:
The class files are kept in the parallel package. If the feature files are kept in the parallel folder these are picked up automatically without adding any configuration to the CucumberOptions annotation.